resource_obmcs_objectstorage_bucket.go

```diff
@@ -86,6 +87,12 @@ func (s *BucketResourceCrud) Create() (e error) {
 		metadata := resourceObjectStorageMapToMetadata(rawMetadata.(map[string]interface{}))
 		opts.Metadata = metadata
 	}
+	if accessType, ok := s.D.GetOk("accessType"); ok {
```

Comment:
it's usually better to skip these bits, depend on a default in the schema and end up with something like:

```go
opts.AccessType = baremetal.BucketAccessType(s.D.Get("accessType"))
```